scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: hosts
+  gather_facts: False
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    image_name: "SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud"
+    stack_name: "caasp-stack"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{ caasp_image_url }}"
+      dest: "{{ caasp_image_path }}"
+    register: _download_caasp_image
+    until: _download_caasp_image | success
+    retries: 5
+    delay: 2
+
+  - name: Upload CaaSP image to glance
+    always_run: True
+    shell: |
+      source ~ardana/service.osrc
+      if ! openstack image list | grep "{{ image_name }}"; then openstack image create --public --disk-format qcow2 --container-format bare --file "{{ caasp_image_path }}" "{{ image_name }}"; fi
+
+  - name: Create heat stack for CaaSP deployment
+    always_run: True
+    shell: |
+      source ~ardana/service.osrc
+      if openstack stack list | grep "{{ stack_name }}"; then openstack stack delete "{{ stack_name}}"; fi
```

Comment:
if the stack delete fails, should we still continue and create the stack, or should we fail? To change that behavior, think of set flags.

In that case, you should also have args:  executable: /bin/bash